### PR TITLE
Stats: Move info-panel to component directory

### PR DESCRIPTION
**Pull request by alternatekev**
This PR moves the `stats/info-panel.jsx` file to `stats/info-panel/index.jsx`. No SCSS was moved for this component. Pinging @timmyc

### CODE REVIEW

**Comment by timmyc** [comment]
@alternatekev I'm getting linter warnings where you have changed double quotes to single quotes - as it is triggering the `jsx-quotes' rule.  Do yo happen to have something that is changing that by default when you update the files?

File: client/my-sites/stats/info-panel/index.jsx [hunk 1]
```diff
@@ -19,26 +19,25 @@ module.exports = React.createClass( {
...
 
-		switch( this.props.module ) {
+		switch ( this.props.module ) {
 
 			case 'referrers':
 				infoView = (
 					<div className={ infoPanelClass }>
 						<p>{ this.translate( 'Learn more about your site’s visibility by looking at the websites and search engines that send the most traffic your way.' ) }</p>
-						<ul className="documentation">
+						<ul className='documentation'>
```
Comment (timmyc) [inline client/my-sites/stats/info-panel/index.jsx]: Changes like this are throwing a lint warning for me now.

Comment (alternatekev) [inline client/my-sites/stats/info-panel/index.jsx]: Just a follow-up comment to note that we discussed these in slack and we're going to be taking care of these as they come up as well as some es6 formatting. I had overzealously fixed some double-quote warnings I was getting so now we're getting single-quote warnings :)

**Comment by alternatekev** [comment]
I'm pretty sure this one is ready to go :+1:

**Pull request merged**